CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ### Draft
 
+-   feat: STRF-9741 Verbose error logging in Stencil CLI by default ([x](https://github.com/bigcommerce/stencil-cli/pull/x))
```

Comment:
can you add a line on how to disable it?

Reply:
sure